core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/checkout/service/workflow/CompleteOrderActivity.java

```diff
@@ -57,9 +50,6 @@ public CompleteOrderActivity(@Qualifier("blCompleteOrderRollbackHandler") Comple
         seed.getOrder().setOrderNumber(determineOrderNumber(seed.getOrder()));
         seed.getOrder().setSubmitDate(determineSubmitDate(seed.getOrder()));
 
-        OrderSubmittedEvent event = new OrderSubmittedEvent(seed.getOrder().getId(), seed.getOrder().getOrderNumber());
```

Comment:
Just make sure that this still works with indexing Orders and Fulfillment Orders in SOLR